test/integration/pods/pods_test.go

```diff
@@ -155,7 +157,9 @@ func TestPodReadOnlyFilesystem(t *testing.T) {
 	ns := framework.CreateTestingNamespace("pod-readonly-root", s, t)
 	defer framework.DeleteTestingNamespace(ns, s, t)
 
-	client := client.NewOrDie(&restclient.Config{Host: s.URL, ContentConfig: restclient.ContentConfig{GroupVersion: testapi.Default.GroupVersion()}})
```

Comment:
Hi there, i tried all that including contacting github, even now while tying this message in the right tool bar it asks me to subscribe "Subscribe, you're not receiving notifications from this thread". I was opening that whoever opened this project to begin with could remove me.